test/e2e/storage/testsuites/provisioning.go

```diff
@@ -147,16 +152,19 @@ func (p *provisioningTestResource) setupResource(driver TestDriver, pattern test
 	// Setup provisioningTest resource
 	switch pattern.VolType {
 	case testpatterns.DynamicPV:
-		if dDriver, ok := driver.(DynamicPVTestDriver); ok {
-			p.sc = dDriver.GetDynamicProvisionStorageClass("")
+		if dynamicDriver, ok := driver.(DynamicPVTestDriver); ok {
+			p.sc = dynamicDriver.GetDynamicProvisionStorageClass("")
 			if p.sc == nil {
 				framework.Skipf("Driver %q does not define Dynamic Provision StorageClass - skipping", driver.GetDriverInfo().Name)
 			}
 			p.driver = driver
-			p.claimSize = dDriver.GetClaimSize()
+			p.claimSize = dynamicDriver.GetClaimSize()
```

Comment:
That's not what I meant - `dDriver` is fine here, albeit a bit brief and it's not obvious what it really stands for.